src/test/ui/async-await/dyn-star-trait-const.rs

```diff
@@ -0,0 +1,14 @@
+// run-pass
```

Comment:
Can you move these to src/test/ui/dyn-star?

Reply:
Done